.circleci/config.yml

```diff
@@ -18,8 +18,11 @@ jobs:
       - run:
           name: 'pylint'
           command: |
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox dev_env.sh
+            source dev_env.sh
             source /usr/local/share/virtualenvs/tap-google-analytics/bin/activate
-            pylint tap_google_analytics --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+            # BUG | https://jira.talendforge.org/browse/TDL-15395
+            pylint tap_google_analytics --disable '{$PYLINT_DISABLE_LIST},line-too-long, fixme, missing-module-docstring, missing-function-docstring, too-many-locals, invalid-name, broad-except, missing-class-docstring, too-many-arguments, no-else-return'
```

Comment:
Interesting. I thought this would have needed double quotes

Reply:
It was single quotes before

Reply:
I just don't understand bash 🤷🏻‍♂️

Reply:
You were correct. Double quotes are needed else we won't be using the variable properly in the pylint call.